integration/tests/karpenter/karpenter_test.go

```diff
@@ -26,34 +28,25 @@ func init() {
 	if err := api.Register(); err != nil {
 		panic(fmt.Errorf("unexpected error registering API scheme: %w", err))
 	}
-	params = tests.NewParams("")
+	params = tests.NewParams("karp")
```

Comment:
I believe I had to do it like this, otherwise it was applying a longer thing. That's why there is the before each thingy that you removed. But it looks like if this works, than it's nice. :)

Reply:
this is defo shorter than other names that also produce long CF stacks. Lets keep it like this for now and see if any failures occur?